consensus/src/sync/live/block_queue/block_request_component.rs

```diff
@@ -19,7 +19,7 @@ use crate::{
 
 #[derive(Debug)]
 pub enum BlockRequestComponentEvent {
-    ReceivedBlocks(Vec<Block>),
+    ReceivedBlocks(u32, Blake2bHash, Vec<Block>),
```

Comment:
I'd add some `rustdoc` documentation, specifically for what we expect on the tuple. Something like:
_Received blocks for a target block number and block hash_